File: Practica_1/src/gestorAplicacion/noPersonas/Bebida.java
package gestorAplicacion.noPersonas;

public class Bebida {

	protected int codigo;
	protected float precio;
	
	public Bebida(int codigo, float precio) {
		this.codigo = codigo;
		this.precio = precio;
	}
	
	public String toString() {
		return "La bebida preparada tiene un precio de " + this.precio;
	}

	public int getCodigo() {
		return codigo;
	}

	public float getPrecio() {
		return precio;
	}

	public void setCodigo(int codigo) {
		this.codigo = codigo;
	}

	public void setPrecio(float precio) {
		this.precio = precio;
	}

	public void fueVendido() {
	}
	
	
}

File: Practica_2/noPersonas/bebida.py
class Bebida:
    
    def __init__(self, codigo, precio):
        self._codigo = codigo
        self._precio = precio
    
    def __str__(self):
        return 'La bebida preparada tiene un precio de ' + self.precio
        
    def getCodigo(self):
        return self._codigo
    
    def getPrecio(self):
        return self._precio
    
    def setCodigo(self, codigo):
        self._codigo = codigo
    
    def setPrecio(self, precio):
        self._precio = precio
    
    def fueVendido():
        pass